src/streaming/protection/controllers/ProtectionController.js

```diff
@@ -539,9 +539,6 @@ function ProtectionController(config) {
         const updateHeaders = function (headers) {
             if (headers) {
                 for (const key in headers) {
-                    if ('authorization' === key.toLowerCase()) {
```

Comment:
This seems sensible, but I think it's worth documenting in the release notes or similar that this behaviour has changed and explicitly setting `protData.withCredentials` to `true` is now required to achieve this so that no one is surprised.

Reply:
Why if we keep this line to ensure backward compatibility? So, by default xhr.withCredentials is set to true when there is an authorization header, but it can be overwritten using protData (```protData.withCredentials=false;```).